timed/projects/views.py

```diff
@@ -17,15 +27,29 @@ class CustomerViewSet(ReadOnlyModelViewSet):
     def get_queryset(self):
         """Prefetch related data.
 
+        If an employee is external, get only assigned customers.
+
         :return: The customers
         :rtype:  QuerySet
         """
-        return models.Customer.objects.prefetch_related("projects")
+        user = self.request.user
+        current_employment = Employment.objects.get_at(user=user, date=date.today())
+
+        if not current_employment.is_external:  # pragma: no cover
+            return models.Customer.objects.prefetch_related("projects")
```

Comment:
The prefetch should also happen for external users. This should be done outside the if/else

Reply:
Ah yes, makes sense